rpcs3/Crypto/decrypt_binaries.cpp

```diff
@@ -11,34 +11,67 @@
 
 LOG_CHANNEL(dec_log, "DECRYPT");
 
-void decrypt_sprx_libraries(std::vector<std::string> modules, std::function<std::string(std::string old_path, std::string path, bool tried)> input_cb)
+usz decrypt_sprx_libraries(std::vector<std::string> modules, usz modules_index, std::string klic_input, std::shared_ptr<void>& klics_storage)
```

Comment:
why is this a void pointer if you're only gonna use u128 anyway?

Reply:
I mean, in the case someone changes the data or the container type in the future, he won't have to edit caller code or expose new private structure of data because it can all be managed in one place.

Reply:
In my opinion this argument makes no sense at all.
By that logic all arguments in Rpcs3 would be void pointers.

Reply:
The purpose of void pointer is to hinder the use of it from code which should not be aware of it,this is similar to what the private keyword does. Here it is used for managing internal function data that is completely irrelevant to the callers.

Reply:
Then this should be wrapped in a class in the first place.

Reply:
Fixed.